scripts/gen_travis.py

```diff
@@ -318,8 +319,10 @@ def main():
         generate_linux(AMD64),
         # PPC tests on travis has been down for a while, disable it for now.
         # generate_linux(PPC64LE),
+        generate_linux(ARM64),
```

Comment:
It looks only this is taking effect, i.e. arm64 on Linux with 13 new tests added. Maybe remove the other two unsupported platforms.

Also it looks 32-bit is not supported:
`Package gcc-multilib is not available, but is referred to by another package.`

So you may want to avoid that combination as well.